easybuild/easyblocks/t/tensorflow.py

```diff
@@ -390,7 +403,9 @@ def get_system_libs(self):
 
         for pkg_name, tf_name in sorted(python_mapping.items(), key=lambda i: i[0].lower()):
             if self.python_pkg_exists(pkg_name):
-                system_libs.append(tf_name)
+                # If it is in deps_with_python_pkg we already added it
+                if tf_name not in deps_with_python_pkg:
+                    system_libs.append(tf_name)
```

Comment:
FTR: Here a set could be used and the check removed. However this approach might be able to catch mistakes when a tf_name is added multiple times although it shouldn't

Reply:
:+1: